libs/core/algorithms/include/hpx/algorithms/traits/pointer_category.hpp

```diff
@@ -25,7 +26,11 @@ namespace hpx::traits {
     {
     };
 
-    struct relocatable_pointer_tag : general_pointer_tag
+    struct trivially_relocatable_pointer_tag : general_pointer_tag
```

Comment:
Where is this used?

Reply:
ATM nowhere, but I need to add this in order to modify `transfer.hpp` to recognize trivial relocatability.